### ISSUE TITLE
Arista - unrecognized ACL syntax

### ISSUE THREAD

**Issue by floatingstatic** [batfish/batfish#3336]
I get the following in the batfish diagnostics log for this ACL syntax from an Arista EOS device:
```
                {
                    "Comment": "This syntax is unrecognized",
                    "Filename": "sw1",
                    "Line": 1456,
                    "Parser_Context": "[extended_access_list_stanza stanza cisco_configuration]",
                    "Text": "33030 deny tcp any 102.66.49.113/24 neq www https syn"
                },
```

A simplified and sanitized version of the configuration I am passing to batfish is as follows:

```
!
hostname sw1
!
int et1
  ip address 192.168.0.1/24
  ip access-group testing123 in
  no shutdown
!
ip access-list testing123
   10 permit tcp host 172.16.1.1 any eq ssh
   20 deny tcp any 192.168.0.0/24 neq www https syn
   30 permit tcp any 192.168.0.0/24
!
```

I am trying to test ACLs that use this syntax, but as a result of lines like this being unrecognized batfish thinks that SSH is permitted through ACL. I am testing with the following bit of code:

```
from pybatfish.client.commands import *
from pybatfish.question.question import load_questions
from pybatfish.datamodel.flow import HeaderConstraints


def main():
    bf_session.coordinatorHost = "localhost"
    bf_init_snapshot('./data/testnet', name='testnet')
    load_questions()
    ssh_flow = HeaderConstraints(srcIps="10.0.0.1", dstIps="192.168.0.0/24", applications=["ssh"])
    try:
        assert_filter_denies(filter_name="testing123", headers=ssh_flow)
    except Exception as e:
        print(e)

if __name__ == "__main__":
    main()
```

Packets sourced from 10.0.0.1 to TCP port 22 (ssh) destined to 192.168.0.0/24 are dropped by sequence 20 in the ACL but `assert_filter_denies()` does not pick this up as a result of batfish ignoring seq 20:

```
Found a flow that was permitted, when expected to be denied
             Node Filter_Name                                               Flow  Action                          Line_Content                                              Trace
0  sw1    testing123  start=sw1 [10.0.0.1:0->192.168...  PERMIT  30 permit tcp any 192.168.0.0/24  Flow permitted by extended ipv4 access-list na...
```

**Comment by progwriter** [comment]
@floatingstatic The unrecognized syntax will be fixed after #3333 is merged.
I will note, however, that since your ACL examines the SYN bit, you will need to explicitly specify that in your query, like so:
```
from pybatfish.datamodel.flow import *

...

ssh_flow = HeaderConstraints(
    srcIps="10.0.0.1", dstIps="192.168.0.0/24", 
    applications=["ssh"], 
    tcpFlags=[MatchTcpFlags(tcpFlags=TcpFlags(syn=True), useSyn=True)]
)
```
Then the flow will be denied.
I know the tcp flag specification is a little cumbersome, we're working on improving it.

**Comment by floatingstatic** [comment]
Excellent, thank you for the tcpFlags example... I had struggled figuring out the correct incantation for that in the docs! Thanks again!
